jib-core/src/main/java/com/google/cloud/tools/jib/frontend/HelpfulSuggestions.java

```diff
@@ -108,6 +112,11 @@ public String forDockerNotInstalled() {
     return suggest("make sure Docker is installed and you have correct privileges to run it");
   }
 
+  public String forInsecureRegistry() {
```

Comment:
This seems like the more useful message, but I don't see a reference to this?

Reply:
This one was already here, just moved it because it was all the way at the bottom of the file. Is this actually more useful? I thought the problem happened even if you set `allowInsecureRegistries`.

Reply:
It's called slightly further down in [BuildStepRunner](https://github.com/GoogleContainerTools/jib/pull/613/files#diff-644d63cbab71bf85ab42dba1ce499fe1R254)

Reply:
Yeah, I think that's the one that happens if `allowInsecureRegistries` isn't set and it tries to do an HTTP fallback. But the error we're throwing here is when HTTP fallback proceeds, but credentials are required.

Reply:
Ah right, I should read more before commenting haha.